modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,36 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, `Role` object, Subscription. These supplement or override defaults generated by `run` or `cleanup`.
+
+|`--install-mode` (string)
+|The `OperatorGroup` is created with the specified `InstallMode`. Format: `InstallModeType[=ns1,ns2[, ...]]`.
+
 |`--kubeconfig` (string)
-|The file path to a Kubernetes configuration file. Default: `$HOME/.kube/config`
+|The file path to a Kubernetes configuration file. Defaults to the location specified by `$KUBECONFIG`, or to default file rules if the environment variable not set.
 
-|`--local`
-|The Operator is run locally by building the Operator binary with the ability to access a Kubernetes cluster using a `kubeconfig` file.
+|`--olm-namespace` (string)
+|The namespace where the OLM is installed. Default: `olm`.
 
-|`--namespace` (string)
-|The namespace where the Operator watches for changes. Default: `default`
+|`--operator-namespace` (string)
+|The namespace where the Operator resources are created. The namespace must already exist in the cluster, or be defined in a manifest that is passed to `--include`.
 
-|`--operator-flags`
-|Flags that the local Operator might require. Example: `--flag1 value1 --flag2=value2`. For use with the `--local` flag only.
+|`--operator-version`
+|The version of the operator to be deployed.
```

Comment:
Please initial cap Operator.

Consider "...to deploy." for a slightly more direct description.